x-pack/plugins/fleet/server/types/models/package_policy.ts

```diff
@@ -111,6 +111,25 @@ const PackagePolicyBaseSchema = {
   output_id: schema.maybe(schema.string()),
   inputs: schema.arrayOf(schema.object(PackagePolicyInputsSchema)),
   vars: schema.maybe(ConfigRecordSchema),
+  overrides: schema.maybe(
+    schema.nullable(
+      schema.object({
+        inputs: schema.maybe(
+          schema.recordOf(schema.string(), schema.any(), {
+            validate: (val) => {
+              if (
+                Object.keys(val).some(
+                  (key) => key.match(/^compiled_inputs(\.)?/) || key.match(/^compiled_stream(\.)?/)
+                )
+              ) {
+                return 'Overrides not allowed';
```

Comment:
should the error message be more specific to say that `complied_inputs` and `compiled_stream` overrides are not allowed?